python/dgl/graphbolt/internal/utils.py

```diff
@@ -135,6 +135,7 @@ def get_attributes(_obj) -> list:
         attribute
         for attribute in dir(_obj)
         if not attribute.startswith("__")
+        and not isinstance(getattr(type(_obj), attribute), property)
```

Comment:
we'd better list what kind of `attributes` want to be obtained from this API in the docstring. and I am wondering the name `get_attributes` is intuitive enough to let user know what it returns.
BTW, do we need to such logic [here](https://github.com/dmlc/dgl/blob/b68a2fc0693c35d9ac0f6eb2c765003d0c96d24e/python/dgl/graphbolt/minibatch.py#L570-L572) if this change is made?

Reply:
@yxy235 could you confirm this? Be sure the exclusion of `property` will not incur other regression or unexpected behavior.

Reply:
I have been investigating that right now. That is why made it a draft PR.

Reply:
get_attributes is used elsewhere. The tests do not pass right now, will work on a fix for that too.

Reply:
The tests pass now.

Reply:
We don't need the exception logic so I removed it.